site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -816,6 +816,17 @@ components:
                   property is ignored.
                   Otherwise, if the `public-key` property is
                   used, this property is mandatory.
+            "subject-dn-bytes":
```

Comment:
can you please set the version of the whole document to 2.6.0 to reflect the fact that we have added new (optional) functionality?

Reply:
I found just this version in the document, so I can raise it to 1.13.0. https://github.com/eclipse-hono/hono/blob/ddebeae9ce4bd3cf03649b3d48ef4d8fd58501f3/site/documentation/content/api/management/device-registry-v1.yaml#L39

Reply:
I found just this version in the document, so I can raise it to 1.13.0. https://github.com/eclipse-hono/hono/blob/ddebeae9ce4bd3cf03649b3d48ef4d8fd58501f3/site/documentation/content/api/management/device-registry-v1.yaml#L39

Reply:
You're right, my bad, we should not increase the major version for this backwards compatible change :+1: